### ISSUE TITLE
setUnit function

### ISSUE THREAD

**Issue by Fredrik1Haider** [OpenModelica/OMSimulator#1141]
Hi!

Currently converting .fmu to an .ssp and this requires some edits to the .ssv file which I export using the OMS api. In addition to changing the value (by setReal) it would be neat if I could also add a unit type (meters, seconds etc.). Currently, I export the .ssv and then using the python etree.xml package go through it and add the units where applicable, however if I could edit the values and add the units before .ssv export that would be even better!

In short: setUnit function to add unit type to connections

**Comment by lochel** [comment]
@Fredrik1Haider, I think we can do two things to improve units in OMSimulator:

- [ ] fetch unit information from the modelDescription file if nothing else is specified
- [ ] add a function oms_setUnit(<cref>, <unit>) to override the unit information in the ssv file

**Comment by Fredrik1Haider** [comment]
@lochel  Sounds perfect!

**Comment by arun3688** [comment]
@Fredrik1Haider The feature is implemented now with this commit  https://github.com/OpenModelica/OMSimulator/commit/f92c2ae0e3f4c0d13c9819af3d63c3126876b883 , an 
 example of how to set the units is available here https://github.com/OpenModelica/OMSimulator/blob/master/testsuite/api/setUnits3.py

**Comment by lochel** [comment]
I just updated the pip package: OMSimulator 2.1.1.post157 contains the changes mentioned by @arun3688.
